sql/updates/world/2015_MM_DD_NN_world.sql

```diff
@@ -0,0 +1,34 @@
+-- Manual Spawn Randy Whizzlesprocket
+SET @GUID := 565;
+DELETE FROM `creature` WHERE `guid` = @GUID;
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`) VALUES
+(@GUID, 17288, 540, 0, 0, 2, 1, 0, 0, 131.106, 254.520, -45.236, 3.951, 7200, 0, 0, 6104, 0, 0, 0, 0, 0);
+
+-- Missing 2xEmblem of Justice from quest 'Imprisoned in the citadel'
+UPDATE `quest_template` SET `RewardItemId1` = 29434, `RewardItemCount1` = 2 WHERE `Id` IN (9524,9525);
```

Comment:
Spawn pos from https://github.com/mangosone/server/blob/master/src/scripts/scripts/outland/hellfire_citadel/shattered_halls/shattered_halls.h

No sniffs of the npc because quest (and npc) removed in cata.
Badges of Justice reward confirmed in wowhead (http://www.wowhead.com/quest=9525/imprisoned-in-the-citadel#comments:id=70960:reply=35374)